Projet-Web-Avance-A4/Technique
# Add log download

**How to test ?**

After doing the related PR in Developpeur, you will connect to your account, then go to Logs.
You will see 2 types of logs :
- Connexion
- Téléchargement de {filename}
